selenium_tests/class_roster/page_objects/class_roster_main_page.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file models the Class Roster Main (Landing) Page
+"""
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.class_roster.page_objects.class_roster_base_page_object \
+    import ClassRosterBasePageObject
+
+
+class Locators(object):
+    CLASS_ROSTER_BREADCRUMB = (By.XPATH, './/h1[contains(., "Class Roster")]')
+
+    @classmethod
+    def FIND_LINK_TEXT(cls, text_value):
+        """
+        Locates the link based on partial link text, given the text value
+        """
+        return By.PARTIAL_LINK_TEXT, "{}".format(text_value)
+
+
+class MainPageObject(ClassRosterBasePageObject):
+    page_loaded_locator = Locators.CLASS_ROSTER_BREADCRUMB
+
+    def get_link_url(self, text_value):
+        """
+        Returns the URL from the partial link text passed in from tests.
```

Comment:
Maybe a little more descriptive? E.g. about what the links are/represent/do in the context of the page and workflow, and why you'd want to get the URL instead of clicking on them.